src/index.ts

```diff
@@ -1,12 +1,12 @@
 import './util/androidFetch'
 
 export { LoginUiProvider } from './components/publicApi/LoginUiProvider'
-export * from './components/publicApi/PublicChangePasswordScreen'
-export * from './components/publicApi/PublicChangePinScreen'
-export * from './components/publicApi/PublicChangeRecoveryScreen'
-export * from './components/publicApi/PublicLoginScreen'
-export * from './components/publicApi/PublicOtpRepairScreen'
-export * from './components/publicApi/PublicSecurityAlertsScreen'
+export * from './components/publicApi/ChangePasswordScreen'
+export * from './components/publicApi/ChangePinScreen'
+export * from './components/publicApi/ChangeRecoveryScreen'
+export * from './components/publicApi/LoginScreen'
+export * from './components/publicApi/OtpRepairScreen'
+export * from './components/publicApi/SecurityAlertsScreen'
```

Comment:
This means we have two files with the same names but in different folders, which is a big no-no. 

If you want to move forward with this commit, you will have to rename the internal scenes to be different, and give them different file names as well. Maybe something like `InternalChangePasswordScreen.tsx`.

Reply:
It’s not an intentional distinction, and kinda confusing! But I guess it’s enoug for now. Carry on.